### PR TITLE
Issue getting "skipped" result to post to TestRail 

### LINKED ISSUE #35: Issue getting "skipped" result to post to TestRail
Hi all,

I'm having issues getting a "skipped" pytest result to post to TestRail. pytest shows the status as "SKIPPED" but nothing is posted to TestRail. "Passed" and "failed" work just fine for me, but not any custom status such as "skipped".

This is the method I'm skipping:

```python
@pytestrail.case('C1')
@pytest.mark.skip()
def test_C1():
    # test code here
```

In my TestRail instance, I have created a custom status with Label = Skipped, status code/id = 6, and System Name = skipped. It is marked as a final status and is an active status. 
screen shot 2018-01-18 at 11 17 01 am

Since the status code is different from the one hardcoded in pytest_testrail/plugin.py (it's hardcoded to 2, which in my TestRail instance is mapped to the "blocked" status), I edited (in my fork of this project) the dictionary PYTEST_TO_TESTRAIL_STATUS in pytest_testrail/plugin.py to:

PYTEST_TO_TESTRAIL_STATUS = {
    "passed": 1,
    "failed": 5,
    "skipped": 6,
} 

I'm not getting any errors, but it's still not posting anything to TestRail. My TestRail version is 5.4.1.3668. 

EDIT: did a few more tests and discovered that it works if I skip the test like this:
```python
@pytestrail.case('C1')
def test_C1():
    pytest.skip()
```

...but not if I skip the test using the @pytest.mark.skip() marker.

### PR DESCRIPTION
**Pull request by zeburek**
Fixed #35

### CODE REVIEW

**Comment by zeburek** [comment]
@allankp Created new MR for bugfix

File: pytest_testrail/plugin.py [hunk 1]
```diff
@@ -182,7 +182,18 @@ def pytest_runtest_makereport(self, item, call):
         """ Collect result and associated testcases (TestRail) of an execution """
         outcome = yield
         rep = outcome.get_result()
-        if item.get_marker(TESTRAIL_PREFIX):
+        if item.get_marker('skip') and item.get_marker(TESTRAIL_PREFIX):
+            testcaseids = item.get_marker(TESTRAIL_PREFIX).kwargs.get('ids')
+            print(rep.ret)
+            if testcaseids:
```
Comment (allankp) [inline pytest_testrail/plugin.py]: Can't see how this would ever be `False`. please explain

File: pytest_testrail/plugin.py [hunk 2]
```diff
@@ -182,7 +182,18 @@ def pytest_runtest_makereport(self, item, call):
...
+                self.add_result(
+                    clean_test_ids(testcaseids),
+                    get_test_outcome('skipped'),
+                    comment=item.get_marker('skip').kwargs.get('reason'),
```
Comment (allankp) [inline pytest_testrail/plugin.py]: What happens here if the `reason` arg is not present ?

Comment (allankp) [reply pytest_testrail/plugin.py]: Might be worth adding a test for this if we add logic to check for it.

**Comment by allankp** [comment]
Closing due to no activity , please reopen if you wish to keep working on this.

**Pull request closed without merging**